Github Action broken in Plone 5.2 on Windows

The error is occurring:

```bash
Getting distribution for 'zope.browser==2.3'.
While:
  Installing.
  Getting section instance.
  Initializing section instance.
  Installing recipe plone.recipe.zope2instance.
  Getting distribution for 'zope.browser==2.3'.

An internal error occurred due to a bug in either zc.buildout or in a
recipe being used:
Traceback (most recent call last):
  File "C:\hostedtoolcache\windows\Python\3.6.8\x64\lib\site-packages\zc\buildout\buildout.py", line 2174, in main
    getattr(buildout, command)(args)
  File "C:\hostedtoolcache\windows\Python\3.6.8\x64\lib\site-packages\zc\buildout\buildout.py", line 701, in install
    [self[part]['recipe'] for part in install_parts]
  File "C:\hostedtoolcache\windows\Python\3.6.8\x64\lib\site-packages\zc\buildout\buildout.py", line 701, in <listcomp>
    [self[part]['recipe'] for part in install_parts]
  File "C:\hostedtoolcache\windows\Python\3.6.8\x64\lib\site-packages\zc\buildout\buildout.py", line 1324, in __getitem__
    options._initialize()
  File "C:\hostedtoolcache\windows\Python\3.6.8\x64\lib\site-packages\zc\buildout\buildout.py", line 1432, in _initialize
    self.initialize()
  File "C:\hostedtoolcache\windows\Python\3.6.8\x64\lib\site-packages\zc\buildout\buildout.py", line 1438, in initialize
    recipe_class = _install_and_load(reqs, 'zc.buildout', entry, buildout)
  File "C:\hostedtoolcache\windows\Python\3.6.8\x64\lib\site-packages\zc\buildout\buildout.py", line 13[88](https://github.com/plone/buildout.coredev/runs/5217597891?check_suite_focus=true#step:6:88), in _install_and_load
    allow_hosts=buildout._allow_hosts
  File "C:\hostedtoolcache\windows\Python\3.6.8\x64\lib\site-packages\zc\buildout\easy_install.py", line [95](https://github.com/plone/buildout.coredev/runs/5217597891?check_suite_focus=true#step:6:95)7, in install
    return installer.install(specs, working_set)
  File "C:\hostedtoolcache\windows\Python\3.6.8\x64\lib\site-packages\zc\buildout\easy_install.py", line 730, in install
    for dist in self._get_dist(req, ws):
  File "C:\hostedtoolcache\windows\Python\3.6.8\x64\lib\site-packages\zc\buildout\easy_install.py", line 574, in _get_dist
    dists = [_move_to_eggs_dir_and_compile(dist, self._dest)]
  File "C:\hostedtoolcache\windows\Python\3.6.8\x64\lib\site-packages\zc\buildout\easy_install.py", line 1750, in _move_to_eggs_dir_and_compile
    os.rename(tmp_loc, newloc)
FileExistsError: [WinError 183] Cannot create a file when that file already exists: 'd:\\a\\buildout.coredev\\buildout.coredev\\eggs\\tmpyw2aj8tu\\zope.browser-2.3-py3.6.egg' -> 'd:\\a\\buildout.coredev\\buildout.coredev\\eggs\\zope.browser-2.3-py3.6.egg'
Error: Process completed with exit code 1.
```

See: https://github.com/plone/buildout.coredev/runs/4305672848?check_suite_focus=true

Is it some incompatibility of buildout 2 with Windows?

Can we remove Windows from GA while this issue is not resolved?

--- a/.github/workflows/run-buildout.yml
+++ b/.github/workflows/run-buildout.yml
@@ -38,6 +38,9 @@ jobs:
           eggs
         key: eggs-${{ hashFiles('**/*.cfg') }}
         restore-keys: eggs-
+      # XXX: buildout fails on Windows when it has cache of eggs. See:
+      # https://github.com/plone/buildout.coredev/issues/772
+      if: matrix.os != 'windows-latest'
     - name: Run buildout
       run: |
         buildout

